src/DSharp.Compiler/Compiler/MetadataBuilder.cs

```diff
@@ -867,6 +867,11 @@ private MethodSymbol BuildMethod(MethodDeclarationNode methodNode, TypeSymbol ty
                 {
                     method = new MethodSymbol(methodNode.Name, typeSymbol, returnTypeSymbol, methodNode.IsExensionMethod);
 
+                    if (methodNode.Attributes.Cast<AttributeNode>().Any(node => node.TypeName.Contains("ScriptIgnoreGenericArguments")))
```

Comment:
Can we move this string to an Internal const or similar in ScriptIgnoreGenericArgumentsAttribute?